src/class-elements/grammar-privatename-no-receiver-error-async-gen-meth.case

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: It is syntax error if IdentifierName is accessed without receiver (Async Generator Method)
+info: |
+  Updated Productions
+
+  ClassElementName :
+    PropertyName
+    PrivateName
+
+  PrivateName ::
+    # IdentifierName
+template: syntax/invalid
+features: [async-iteration, class-methods-private]
+---*/
+
+//- elements
+async * #m() {}
+method() {
+#m();
```

Comment:
the simple get operation should apply for actual methods, no need to use the call parentheses on them. `#m` should be enough to trigger the syntaxerror.